try : Exception trace
catch : Handling the  exception
finally
throws
throw
try with resources


